### PR TITLE
Add GCCH and DoD menu options and screenshot

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8741

### CODE REVIEW

File: Teams/teams-app-permission-policies.md [hunk 1]
```diff
@@ -107,6 +109,15 @@ Use org-wide app settings to control whether users can install third-party apps.
 
 As mentioned earlier, to allow third-party apps, you can either edit and use the global (Org-wide default) policy or create and assign custom policies.
 
+### For GCCH and DoD clouds
+
+1. On the **Permission policies** page, select **Org-wide app settings**. You can then configure the settings you want in the panel.
+
+    ![Screenshot of org-wide app settings in GCCH and DoD.](media/app-permission-policies-gcch-dod-org-wide.png)
+
+1. Under **Blocked apps**, add the apps you want to block across your organization. In a Microsoft 365 Government - GCCH and DoD deployment of Teams, all third-party apps are added to this list by default. When you block an app org-wide, the app is automatically blocked for all your users, regardless of whether it's allowed in any app permission policies
```
Comment (JohanFreelancer9) [inline Teams/teams-app-permission-policies.md:118]:
```suggestion
1. Under **Blocked apps**, add the apps you want to block across your organization. In a Microsoft 365 Government - GCCH and DoD deployment of Teams, all third-party apps are added to this list by default. When you block an app org-wide, the app is automatically blocked for all your users, regardless of whether it's allowed in any app permission policies.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@ashishguptaiitb PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
@ashishguptaiitb is creating a separate PR to add these changes.

**Pull request closed without merging**